patches/@txnlab+use-wallet+3.10.0.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/node_modules/@txnlab/use-wallet/dist/index.cjs b/node_modules/@txnlab/use-wallet/dist/index.cjs
```

Comment:
Your fix has been merged in a released, so we can update and remove this patch. Might be worth a quick test after update to confirm.